packages/evm/src/journal.ts

```diff
@@ -85,6 +97,13 @@ export class Journal {
   }
 
   private touchAccount(address: string) {
+    // If preimages are being reported, add the address to the preimages map
+    if (this.preimages !== undefined) {
+      const bytesAddress = unprefixedHexToBytes(address)
+      const hashedKey = this.stateManager.getAppliedKey(bytesAddress)
```

Comment:
I am confused. We want to store the preimage right? This stores the hashed key? :thinking: